<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="styleSheet">
         <string notr="true">background-color: #2f3e40; /*#2f3e46,1a535c;1d3557;233d4d*/
color: rgb(255, 255, 255);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>9</number>
         </property>
         <property name="topMargin">
          <number>9</number>
         </property>
         <property name="rightMargin">
          <number>9</number>
         </property>
         <property name="bottomMargin">
          <number>9</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>9</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: #2f3e40; /*#2f3e46,1a535c;1d3557;233d4d*/
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Ajout bon de sortie</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>510</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_4">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0" colspan="2">
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: #52796f;</string>
              </property>
              <property name="text">
               <string>Declarant</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLineEdit" name="declarant">
              <property name="styleSheet">
               <string notr="true">border : 1px solid #2f3e46;
border-radius:10px;
padding: 3px;
background-color: white;
color: #2f3e46;
</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3" colspan="2">
             <widget class="QLabel" name="label_5">
              <property name="font">
               <font>
                <family>Segoe UI</family>
                <pointsize>9</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: #52796f;</string>
              </property>
              <property name="text">
               <string>NÂ° Dossier</string>
              </property>
             </widget>
            </item>
            <item row="0" column="5">
             <widget class="QLineEdit" name="num_dossier">
              <property name="styleSheet">
               <string notr="true">border : 1px solid #2f3e46;
border-radius:10px;
padding: 3px;
background-color: white;
color: #2f3e46;
</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_6">
              <property name="font">
               <font>
                <family>Segoe UI</family>
                <pointsize>9</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: #52796f;</string>
              </property>
              <property name="text">
               <string>Date</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1" colspan="2">
             <widget class="QLineEdit" name="date">
              <property name="styleSheet">
               <string notr="true">border : 1px solid #2f3e46;
border-radius:10px;
padding: 3px;
background-color: white;
color: #2f3e46;
</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QLabel" name="label_7">
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: #52796f;</string>
              </property>
              <property name="text">
               <string>Agent</string>
              </property>
             </widget>
            </item>
            <item row="1" column="4" colspan="2">
             <widget class="QLineEdit" name="agent">
              <property name="styleSheet">
               <string notr="true">border : 1px solid #2f3e46;
border-radius:10px;
padding: 3px;
background-color: white;
color: #2f3e46;
</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_4">
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: #52796f;</string>
              </property>
              <property name="text">
               <string>Transit</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QLineEdit" name="transit">
              <property name="styleSheet">
               <string notr="true">border : 1px solid #2f3e46;
border-radius:10px;
padding: 3px;
background-color: white;
color: #2f3e46;
</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_5">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="valider_ajout_bon_sortie">
              <property name="font">
               <font>
                <family>Segoe UI</family>
                <pointsize>9</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {width: 65px;
background-color: white;
border: 2px solid #3bb273;
color: #3bb273;
padding: 6px;
border-radius: 10px;}

QPushButton:hover {background-color: #3bb273;color: white;}
QPushButton:pressed { margin: 1px; }
</string>
              </property>
              <property name="text">
               <string>Valider</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="fermer_ajout_bon_sortie">
              <property name="font">
               <font>
                <family>Segoe UI</family>
                <pointsize>9</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {background-color: white;
border: 2px solid #ee2e31;
color: #ee2e31;
padding: 5px;
border-radius: 10px;}
QPushButton:hover {
background-color: #ee2e31;
color: white;
}
QPushButton:pressed{
margin: 1px;
}
/*#ee2e31*/
</string>
              </property>
              <property name="text">
               <string>Fermer</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
